### DISCUSSION TITLE
Live preview not working in ChromeOS app when offline

### DISCUSSION THREAD

**Discussion by 04474** [phcode-dev/phoenix#1857] [Q&A] [answered]
the file is saved to my local storage i thought it should be able to preview  it has been a while since have used this i don't know about your update also but that is how it does before one the file is saved locally there is no need for internet connection 

![Screenshot 2024-09-18 00 52 12](https://github.com/user-attachments/assets/d3d0f45a-7912-4ef5-924a-8fae47b9970e)
![Screenshot 2024-09-18 00 52 12](https://github.com/user-attachments/assets/63e56847-737e-440a-92d4-dcedc5f44a2a)

**Comment by abose** [comment]
Try closing and reopening the app one or two times. There is a new native app available for chromeos at https://phcode.io id recommended upgrading to that.

**Comment by 04474** [comment]
Nope it's like the web version so I downloaded it on my Chromebook and it
does work well on it while offline,now bith online and offline it doesn't
work I usually get that same error

**Comment by abose** [comment, marked as answer]
After a bit of investigation, it looks like making live preview work when offline in the browser is not possible. Since the current ChromeOS app is a packaged PWA, it will inherit the offline restriction from the browser.

But this should be fixable in the ChromeOS Native app once we move to a more native implementation. Please follow https://github.com/phcode-dev/phoenix/issues/1858 to track the issue in ChromeOS Native apps. This is not likeley to be worked on soon as this affects only online use case and we fully invested in terminal and git support for the near future.

Live preview Currently needa an active internet connection for phcode.dev web app and the ChromeOS native app.
This is not an issue in Windows/Linux/Mac OS native app though.
